app/src/main/scala/io/codegalaxy/app/info/ListItemNavIcon.scala

```diff
@@ -55,7 +58,7 @@ object ListItemNavIcon extends FunctionComponent[ListItemNavIconProps] {
       override val justifyContent = JustifyContent.`flex-end`
     }
     val statsLabel: Style = new TextStyle {
-      override val color = Color.dodgerblue
```

Comment:
FYI: the idea to use `Color.dodgerblue` was to make it more prominent and look like a button, which user can press and go inside, which is what essentially is happening.